src/test/java/unit/uk/gov/ida/verifyserviceprovider/services/EidasAssertionServiceTest.java

```diff
@@ -120,7 +113,7 @@ public void shouldCallValidatorsCorrectly() {
         List<Assertion> assertions = asList(
             anAssertionWithAuthnStatement(EIDAS_LOA_HIGH, "requestId").buildUnencrypted());
 
-        eidasAssertionService.translateSuccessResponse(assertions, "requestId", LevelOfAssurance.LEVEL_2, null);
+        eidasAssertionService.translateSuccessResponse(assertions, "requestId", LEVEL_2, null);
```

Comment:
I disagree with Codacy that this is an improvement, but it's probably not a battle worth fighting.